async/parsed/parsed.go

```diff
@@ -0,0 +1,35 @@
+// Package parsed just holds the domain types for dealing with the output of the
```

Comment:
Perhaps link to the repo that this relies upon? It's not obvious from this comment that this describes the contours of an external dependency.

Reply:
GC, done